src/pages/journal/journal.model.ts

```diff
@@ -10,6 +11,8 @@ export type TestSlot = {
 
 export type JournalModel = {
   isLoading: boolean,
+  lastRefreshed: Date,
   data: ExaminerWorkSchedule,
+  slots: JournalSlot[],
```

Comment:
The slots are inside of the data already, not sure if we need it as a separate property?

Reply:
Keeping this around for now so we're not throwing away data about the examiner.